.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store) ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png))

Reply:
resolved